README.md

```diff
@@ -192,25 +196,34 @@ druid = "druid:#{ARGV[0]}"
 path = Moab::StorageServices.object_path(druid)
 puts "#{druid} found at #{path}"
 
-moab = Moab::StorageObject.new( druid,  Moab::StorageServices.object_path(druid) )
+moab = Moab::StorageObject.new(druid,  Moab::StorageServices.object_path(druid))
 
 # Validation checks for file existence, but not content, of a well-formed Moab.
 # It does not read files or perform checksum validation.
 object_validator = Stanford::StorageObjectValidator.new(moab)
 validation_errors = object_validator.validation_errors # Returns an array of hashes with error codes
+puts "\nChecking stuctural validition of #{druid}\n"
 if validation_errors.empty?
-  p "Yay! #{moab.digital_object_id} passed validation"
+  puts "\nYay! Latest version of #{moab.digital_object_id} passed structural validation.\n"  
```

Comment:
i'd strike the "Latest version of " part, since i think this structural validation applies to all versions:  https://github.com/sul-dlss/moab-versioning/blob/1cfe59cfb4f9fea7a1cc35b936a8f5a7ddd94029/lib/moab/storage_object_validator.rb#L44-L50